docs/usermanual.md

```diff
@@ -843,6 +842,38 @@ the example case the command will be:
 }
 ```
 
+The agent provides additional information, hereafter referred as **JEXL context**, in order to be used to evaluate the
+expression of the **command** as well as the **device endpoint** when apply. In all cases the following data is available to all expressions within the JEXL context:
+
+-   `id`: device ID
+-   `type`: NGSI entity type (principal)
+-   `entity_name`: Entity Name
+-   `service`: device service (`Fiware-Service`)
+-   `subservice`: device subservice (`Fiware-ServicePath`)
+-   `staticAttributes`: static attributes defined in the device or config group (flatted into context object).
+-   `attribute`: attribute name and value involved in the command.
```

Comment:
```suggestion
-  Static attributes defined in the device or config group (flatted into context object), by name.
-  The attribute involved in the command, by name.
```

I think is better not to use the `code style` in this case, because it is used by the others (`id`, etc.) and the reader may think they are literal references.